roles/filetree_create/templates/current_inventories.j2

```diff
@@ -61,7 +61,7 @@ controller_inventories:
       {{ template_overrides_resources.inventory[current_inventories_asset_value.name].variables
          | default(template_overrides_global.inventory.variables)
          | default(current_inventories_asset_value.variables)
-         | from_yaml | to_nice_yaml(indent=2, sort_keys=False) | indent(width=6, first=False) | replace("'{{", "!unsafe \'{{") | replace("'{%", "!unsafe \'{%") }}
+         | from_yaml | to_nice_yaml(indent=2, sort_keys=False) | indent(width=6, first=False) | regex_replace('(^[^:]*): (.*){{', '\\g<1>: !unsafe \\g<2>{{', multiline=True)}}
```

Comment:
This is taking in consideration the following: `variables: "{{ my_variables }}"`
But is not taking in consideration the following one: `variables: "{% my_variables if (time is sunny) else my_rain %}"`

Could the expression `regex_replace('(^[^:]*): (.*){[{%]', '\\g<1>: !unsafe \\g<2>{{', multiline=True)` work?